x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/get_breadcrumbs_for_page.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ChromeBreadcrumb } from '@kbn/core/public';
+import { SecurityPageName } from '../../../../app/types';
+import { APP_NAME } from '../../../../../common/constants';
+import { getAppLandingUrl } from '../../link_to/redirect_to_landing';
+
+import { GetSecuritySolutionUrl } from '../../link_to';
+import { getAncestorLinksInfo } from '../../../links';
+import { GenericNavRecord } from '../types';
+
+export const getLeadingBreadcrumbsForSecurityPage = (
+  pageName: SecurityPageName,
+  getSecuritySolutionUrl: GetSecuritySolutionUrl,
+  navTabs: GenericNavRecord,
+  isGroupedNavigationEnabled: boolean
+): [ChromeBreadcrumb, ...ChromeBreadcrumb[]] => {
```

Comment:
can it be replaced with?
```suggestion
): ChromeBreadcrumb[] => {
```